internal/pkg/template/templates/workloads/services/static-site/cf.yml

```diff
@@ -222,19 +227,42 @@ Resources:
             Type: Map
             End: true
             ItemsPath: $.GetMappingFile.files
-            ItemSelector:
-              mapping.$: $$.Map.Item.Value
-              destBucket.$: $.DestinationBucket
-              srcBucket.$: $.SourceBucket
             ItemProcessor:
               ProcessorConfig:
                 Mode: INLINE
-              StartAt: CopyFile
+              StartAt: ContentTypeChoice
               States:
+                ContentTypeChoice:
```

Comment:
Why not copy with whatever contenttype we have in the mapping file 🤔

Reply:
if it's not present using `$.contentType` throws an error🥺 and if it's `contentType: ""`, then it uploads with an empty string `Content-Type` instead of the default value (`binary/octet-stream`)

Reply:
Interesting seems like we don't have a good way of specifying "REPLACE" only if `contentType != ""`

Reply:
we could add a new state to do that - my thinking was that we should treat `contentType: ""` and missing content type the same. what do you think?